### PR TITLE
Ft post service testing

### PR DESCRIPTION
**Pull request by josephtomellis**
post service test complete with placeholder post service, url, and authorization dependencies

### CODE REVIEW

File: fakebook/karma.conf.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = function (config) {
         // the possible options are listed at https://jasmine.github.io/api/edge/Configuration.html
         // for example, you can disable the random execution with `random: false`
         // or set a specific seed with `seed: 4321`
+        random: false
```
Comment (yetanothersegfault) [inline fakebook/karma.conf.js]: I would take this out of the PR. We don't need to update the karma.conf.js on the branch.

Here is a link for that
https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

**Review by yetanothersegfault — COMMENTED** [review]
If you are ready for this to go to Nick ask that he review changes on the right hand side.

But I would at least remove the change to karma.conf.js first

File: fakebook/src/app/model/user.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export interface User {
+    id: number;
+    firstName: string;
+    lastName: string;
+    email: string;
+    phoneNumber: string | undefined;
+    password: string;
```
Comment (RevatureGentry) [inline fakebook/src/app/model/user.ts]: Do we need to be sending the user's password from the server?

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
LGTM - the one item I have is, does the client need to know about the user's password? Are we employing Okta here or another auth system?

**Comment by josephtomellis** [comment]
The models have been cleaned up to better reflect their counterparts in the backend services. the password property was removed from the user interface, as authentication is being handled using okta.

**Review by RevatureGentry — APPROVED** [review]
LGTM 👍

**Pull request merged**